compose/wordpress.md

```diff
@@ -52,6 +52,8 @@ Compose to set up and run WordPress. Before starting, make sure you have
          depends_on:
            - db
          image: wordpress:latest
+         volumes:
+           - wordpress_data: /var/www/html
```

Comment:
```suggestion
           - wordpress_data:/var/www/html
```

Reply:
The rest of the file does not have spaces after the colon though...